apps/galaxy/src/modules/auth/SignInForm/SignInForm.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import { ErrorMessage } from "@hookform/error-message";
+import useSignIn from "../../../services/signin/useSignIn";
+import { Button, Container, FormLabel, FormControl, Input } from "@cc/ui-chakra";
+
+interface FormData {
+  email: string;
+  password: string;
+}
+
+const SignInForm = () => {
+  const { isLoading, handleSignInSubmit } = useSignIn();
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+  } = useForm<FormData>();
+  const formSubmit = ({ email, password }: FormData) => {
+    handleSignInSubmit(email, password);
+  };
+  const showError = (obj: { message: string }) => {
+    return <p>{obj.message}</p>;
+  };
+
+  return (
+    <FormControl as="form" width="100%" onSubmit={handleSubmit(formSubmit)}>
+      <FormLabel fontWeight="600">Email</FormLabel>
```

Comment:
I feel like we can wrap this inside an `<Input>` component since we will most definitely be reusing this in other forms.

Reply:
Okay, as per discussion, since we not creating reusable component at this stage, then I guess we can close this comment. 👍